src/status_im/components/list/views.cljs

```diff
@@ -1,23 +1,58 @@
 (ns status-im.components.list.views
-  (:require [status-im.components.react :as rn]
-            [reagent.core :as r]
-            [status-im.components.common.common :as common]
+  (:require [reagent.core :as r]
+            [status-im.components.list.styles :as lst]
+            [status-im.components.react :as rn]
             [status-im.utils.platform :as p]))
 
 (def flat-list-class (rn/get-class "FlatList"))
+(def section-list-class (rn/get-class "SectionList"))
 
 (defn- wrap-render-fn [f]
   (fn [o]
     (let [{:keys [item index separators]} (js->clj o :keywordize-keys true)]
       (r/as-element (f item index separators)))))
 
+(defn- separator []
+  [rn/view lst/separator])
+
+(defn- section-separator []
+  [rn/view lst/section-separator])
+
+(defn base-list-props [render-fn empty-component]
+  (merge {:renderItem   (wrap-render-fn render-fn)
+          :keyExtractor (fn [_ i] i)}
+         (when p/ios? {:ItemSeparatorComponent (fn [] (r/as-element [separator]))})
+         ; TODO Does not work with our current ReactNative version
```

Comment:
Can you write TODO(jeluard) so it's clear who to ask in future in case of problems? Same with TODO below